relatio/utils.py

```diff
@@ -56,7 +56,13 @@ def split_into_sentences(
         time.sleep(1)
         docs = tqdm(docs)
 
+    # Ready to be submitted now and test for merge
     for doc in docs:
+        if not (isinstance(doc["doc"], str)):
+            # Blank sentence (str format)
```

Comment:
Same here, can be removed (not informative)

Reply:
Yeah those were for me, I guess the non useful comments can be removed.